modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTable.scala

```diff
@@ -284,16 +287,51 @@ private sealed trait SequenceTableBuilder[S: Eq, D <: DynamicConfig: Eq]
                 case _                                    =>
                   TagMod.empty
 
-        val (icon, label) =
-          if table.getIsAllRowsExpanded() then (Icons.Minus, "Collapse all visits")
-          else (Icons.Plus, "Expand all visits")
+        def modExpanded(
+          f: (
+            Row[Expandable[HeaderOrRow[SequenceIndexedRow[D]]]]
```

Comment:
There's a type alias defined `SequenceTableRowType = Expandable[HeaderOrRow[SequenceIndexedRow[D]]]` which can be used as a shorthand.

Reply:
Thanks, typing out that long type made me need a vacation!